ShopLens/VoicedText/TextVoicers/ITextVoicer.cs

```diff
@@ -0,0 +1,27 @@
+﻿namespace VoicedText.TextVoicers
+{
+    /// <summary>
+    /// Interface that defines methods needed for a text voicer.
+    /// </summary>
+    public interface ITextVoicer
+    {
+        /// <summary>
+        /// Sets the speed of the text voicer.
+        /// </summary>
+        /// <returns></returns>
+        void SetSpeed(int speedOfVoicer);
```

Comment:
We should perhaps expose `enum VoicerVolume` from TextSpeechSythesizer.cs to allow passing it as a parameter. Integer is really weird when you know it only accepts like 4 values and all other will be exceptions.

Reply:
Then I won't have to cast those enum values to int, right? I didn't even think of this!